.vscode/settings.json

```diff
@@ -7,4 +7,4 @@
 	"[svelte]": {
 		"editor.defaultFormatter": "esbenp.prettier-vscode"
 	}
-}
+}
```

Comment:
there seems to be a change still (see diff), if you can revert this change as well we can merge